docs/examples/gnmi-clients/arista-gnmi/index.md

```diff
@@ -799,59 +799,59 @@ gnmi -addr 10.83.13.139:6030 -username admin get origin=eos_native '/Sysdb/conne
 ##### Configure neighbor address and peer AS
 
 ```shell
-gnmi -addr 10.83.13.139:6030 -username cvpadmin -password arastra \
+gnmi -addr 192.0.2.139:6030 -username cvpadmin -password arista \
   update '/network-instances/network-instance[name=default]/protocols/protocol[name=BGP]/bgp/neighbors/neighbor[neighbor-address=10.10.100.43]' \
```

Comment:
are we concerned with bgp peer addresses?

Reply:
yes, this is in sample CLI.  i’ve updated this. (hopefully)